src/App.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import {fetchWeather} from './utils/api';
+
+class App extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      city: "La Paz",
```

Comment:
This is just for testing right? because we want to focus on `Cochabamba` and its counties/cities/villages

Reply:
Yes, It is. My component app is for testing connectivity